elodie/filesystem.py

```diff
@@ -10,11 +10,33 @@
 import re
 import shutil
 import time
+from os import path
+from configparser import RawConfigParser
 
 from elodie import geolocation
+from elodie import constants
 from elodie import log
 from elodie.localstorage import Db
 
+__DIR__ = None
```

Comment:
In the suggested `config` module perhaps we can have a simple method that loads in `config.ini` and returns it.

```
# config.py
def load():
    if hasattr(load, "config"):
        return load.config
    
    load.config = RawConfigParser().read(config_file)
    return load.config
```

I'm 80% sure something like that would work.